AUTHORS

```diff
@@ -16,7 +16,7 @@ Pascal Obry
 Ger Siemerink
 EdgarLux
 Timur I. Davletshin
-Michel Leblond
```

Comment:
Unwanted change!

Reply:
It's not a my commit: i suppose it's a wrong deletion.